serveradmin/urls.py

```diff
@@ -25,6 +25,9 @@
 ]
 
 for app in settings.INSTALLED_APPS:
+    if '.apps.' in app and app.endswith('Config'):
```

Comment:
The names are unfortunately just convention and can be completely different. If this is fine for us, then ok.
With `apps.get_app_configs()` (https://docs.djangoproject.com/en/1.10/ref/applications/#django.apps.apps.get_app_configs) we could have a more reliable solution though.

Reply:
Neat! Didn't knew that will do.